### ISSUE TITLE
Convert loop to linq expression

### ISSUE THREAD

**Issue by ElemarJR** [code-cracker/code-cracker#22]
We will start with simple ones, looking into foreach and a simple if:

This:

````csharp
var peopleNames = new List<string>();
foreach (var person in people)
{
    peopleNames.Add(person.Name);
}
````

Becomes:

````csharp
var peopleNames = from person in people
                  select person.Name;
````

And this:

````csharp
var peopleNames = new List<string>();
foreach (var person in people)
{
    if (person.Id > 3)
        peopleNames.Add(person.Name);
}
````

Becomes:

````csharp
var peopleNames = from person in people
                  where person.Id > 3
                  select person.Name;
````

Diagnostic id: `CC0050`
Category: `Refactoring`
Severity: `Hidden`

**Comment by nyctef** [comment]
I've actually been playing around with some code to do this -- just came across code-cracker today :)

I'm a Roslyn newbie so the code is pretty terrible, but it might be useful to have a look at: https://github.com/nyctef/ForeachToLinqAnalyzer

**Comment by giggio** [comment]
I am working on it.

**Comment by giggio** [comment]
Closed on #502.
